recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Fetch visionworkbench
+cd $SRC_DIR
+wget https://github.com/visionworkbench/visionworkbench/archive/refs/tags/3.5.1.tar.gz > /dev/null 2>&1 # this is verbose
+tar xzf 3.5.1.tar.gz
+
+echo List all files
+ls -d *
```

Comment:
This is not necessary and there are many commented out lines here that should be removed/cleaned if they do not serve any purpose.

Reply:
Addressed.